src/utils/digitalAssets/tests/addETHAsset.test.js

```diff
@@ -0,0 +1,26 @@
+// @flow strict
+
+import addETHAsset from '../addETHAsset'
```

Comment:
Better to import from `index.js` because it is the public interface you are testing for existence.